Student/Student/Submissions/SubmissionDetails/SubmissionDetailsPresenter.swift

```diff
@@ -65,7 +65,7 @@ class SubmissionDetailsPresenter: PageViewLoggerPresenterProtocol {
 
     var quizzes: Store<GetQuiz>?
 
-    var selectedAttempt: Int = 0
+    var selectedAttempt: Int?
```

Comment:
This is the fix. Defaulting to 0 meant we would select the submission object before there was an attempt. We want this to be nil so that we always select the most recent submission before the user explicitly selects an attempt.